guides/common/modules/proc_installing-proxy-packages.adoc

```diff
@@ -4,7 +4,7 @@
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-update}
+# {client-package-update-el8}
```

Comment:
Why a `client-*` command on SmartProxy?

Reply:
It was a client attribute before; probably because it would not work on Capsule otherwise. Fixed now.

At this point in time, you don't have access to "satellite-maintain packages update" yet AFAIK.